genie-common/src/main/java/com/netflix/genie/common/dto/v4/JobRequest.java

```diff
@@ -53,6 +55,8 @@
     private final JobMetadata metadata;
     @Valid
     private final ExecutionResourceCriteria criteria;
+    @Valid @NotNull
```

Comment:
Pretty sure `@Valid` already applies `@NonNull` but could double check. Also I thought we said `@Valid` on file did nothing?

Reply:
From my understanding, `@Valid` enabled validation and `@NotNull` is what the validation checks for (since there is no file-specific validator.
I'll remove `@Valid`. It's _probably_ superfluous.